cmd/job_refresh.go

```diff
@@ -0,0 +1,167 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/odpf/salt/log"
+	"github.com/pkg/errors"
+	cli "github.com/spf13/cobra"
+	"google.golang.org/grpc"
+
+	pb "github.com/odpf/optimus/api/proto/odpf/optimus/core/v1beta1"
+	"github.com/odpf/optimus/config"
+	"github.com/odpf/optimus/models"
+)
+
+const (
+	refreshTimeout = time.Minute * 15
+)
+
+func jobRefreshCommand(conf *config.ClientConfig) *cli.Command {
+	var (
+		projectName string
+		verbose     bool
+		cmd         = &cli.Command{
+			Use:     "refresh",
+			Short:   "Refresh job deployments",
+			Long:    "Redeploy jobs based on current server state",
+			Example: "optimus job refresh",
+		}
+	)
+
+	cmd.Flags().StringVarP(&projectName, "project", "p", projectName, "Optimus project name")
```

Comment:
looks like we are accepting the flags but it doesn't work, as we are always referring from config. @deryrahman is working on fixing loading of specs from a single place, lets do the same here.

Reply:
yes, it's known issue @sravankorumilli , will be fixed on https://github.com/odpf/optimus/pull/274, let's merge it once the PR is approved

Reply:
for now i have removed the project flag